automation/setup-local-env-for-e2e.sh

```diff
@@ -0,0 +1,425 @@
+    echo "  --bee: The name of an existing BEE environment."
+    echo "  --billingProject: The name of an existing billing project in the given BEE environment. If not specified, a random billing project will be created and attached to the landing zone (optional: the attachLandingZoneToBillingProject method will be skipped, the landing zone must already been attached to the specified billing project)."
+    echo "  --tenantId: Azure tenant ID (optional, default: fad90753-2022-4456-9b0a-c7e5b934e408)."
+    echo "  --subscriptionId: Azure subscription ID (optional, default: f557c728-871d-408c-a28b-eb6b2141a087)."
+    echo "  --mrgId: Azure Managed Resource Group name (optional, default: staticTestingMrg)."
+    echo "  --landingZoneId: Landing Zone ID. An existing LZID tag within a given MRG (optional, default: f41c1a97-179b-4a18-9615-5214d79ba600)."
+    echo ""
+    echo "# Use Cases"
+    echo "# ========="
+    echo "#"
+    echo "# Use Case 1 (Create a random billing project and attach it to default Landing Zone):"
+    echo "#"
+    echo "# $0 --e2eEnv azure_e2e.env --bee rawls-593017852-1-dev"
+    echo "#"
+    echo "# Replace rawls-593017852-1-dev with the name of an existing BEE environment you want your tests to run against"
+    echo "#"
+    echo "# Use Case 2 (Reuse a billing project that already exists in the BEE environment):"
+    echo "#"
+    echo "# $0 --e2eEnv azure_e2e.env --bee rawls-593017852-1-dev --billingProject tmp-billing-project-44302a2c-5"
+    echo "#"
+    echo "# Replace rawls-593017852-1-dev with the name of an existing BEE environment you want your tests to run against"
+    echo "# Replace tmp-billing-project-44302a2c-5 with the name of a valid billing project in the BEE environment"
+    echo "#"
+    echo "# Use Case 3 (Create a random billing project and attach it to new Landing Zone):"
+    echo "#"
+    echo "# $0 --e2eEnv azure_e2e.env --bee rawls-593017852-1-dev \\"
+    echo "#    --tenantId fad90753-2022-4456-9b0a-c7e5b934e408           \\"
+    echo "#    --subscriptionId f557c728-871d-408c-a28b-eb6b2141a087     \\"
+    echo "#    --mrgId staticTestingMrg                                  \\"
+    echo "#    --landingZoneId f41c1a97-179b-4a18-9615-5214d79ba600"
+    echo "#"
+    echo "# Replace rawls-593017852-1-dev with the name of an existing BEE environment you want your tests to run against"
+    echo "# Replace (tenantId, subscriptionId, mrgId, landingZoneId) with new coordinates"
+    echo "# Please refer to https://github.com/broadinstitute/terra-github-workflows/blob/main/.github/workflows/attach-landing-zone-to-bee.yaml"
+    echo "#"
+    echo "# This script generates a \$e2eEnv file. You can run tests locally as follows."
+    echo "#"
+    echo "# source ${TEST_RESOURCES}/\$e2eEnv"
```

Comment:
Or actually, I guess the typo is here?

Typo here-- prints out as source /Users/roberts/Documents/source/rawls/automation/src/test/resources/$e2eEnv.

We could also add a comment about how to configure the environment file in IntelliJ. Perhaps:

"If you are running the test via IntelliJ, in the Run/Debug Configuration use the EnvFile tab to provide the path to the generated .env file."

Reply:
Updated the help section including IntelliJ run/debug config.